boto/connection.py

```diff
@@ -1112,6 +1115,7 @@ def make_request(self, action, params=None, path='/', verb='GET'):
             http_request.params['Action'] = action
         if self.APIVersion:
             http_request.params['Version'] = self.APIVersion
+        statsd.increment('boto.request', tags=["action:%s" % action or 'unknown', "host:"])
```

Comment:
I don't know the answer to that - considering that we can split by host on one dimension, vs role/name on another - would a different set of tags accomplish the same without removing host?